pkg/proxy/ipvs/proxier.go

```diff
@@ -274,6 +274,19 @@ func (r *realIPGetter) NodeIPs() (ips []net.IP, err error) {
 // Proxier implements ProxyProvider
 var _ proxy.ProxyProvider = &Proxier{}
 
+// ParseExcludedCIDRs parses the input strings and returns net.IPNet
+// The validation has been done earlier so the error condition will never happen under normal conditions
+func ParseExcludedCIDRs(excludeCIDRStrs []string) []*net.IPNet {
+	var cidrExclusions []*net.IPNet
+	for _, excludedCIDR := range excludeCIDRStrs {
+		_, n, err := net.ParseCIDR(excludedCIDR)
+		if err == nil {
```

Comment:
can we flip the err check here?

```go
if err != nil {
    continue
}

cidrExclusions = append(cidrExclusions, n)
```

Easier to read IMO 

```

Reply:
Also curious if an err here warrants logging. What do you think?

Reply:
Please see comment on line 278 - there is no error during parsing.

Reply:
@andrewsykim 
Please let me know if the code can be kept as is.

The reason I ask you to confirm was that, I no longer have the branch locally based on which this PR was generated.
If you insist I change any of the code, I need to re-create the PR.

Thanks

Reply:
ah I see, can you open follow-up PR address those comments then?

Reply:
Sure. Once this goes in, I will file follow-on PR.